Assets/Scripts/PowerUps/PowerUpBase.cs

```diff
@@ -19,6 +19,17 @@ void Update()
 
     public void InitializePowerUp()
     {
+        if (interactor != null && interactor is Player player)
+        {
+            PlayPickUpSound();
+            HandlePowerUp(player);
+            Destroy(gameObject);
+        }
+    }
+
+    private void PlayPickUpSound()
+    {
+        AudioManager.m_Instance.PlaySound("MAGIC_Powerup", 1.2f, 1.2f);
         m_TimeOfActivation = Time.time;
```

Comment:
I think this was a merge issue. The lines your pr is trying to add here should have been removed when we added caleb's item pickups.

Reply:
hmmm... interesting...

ok shall fix that nao